CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
   - Proof of Work consensus
   - Fast Sync
 
+- `--host-whitelist` has been deprecated since 2020 and its related option will be removed in a future release.
```

Comment:
can you move this to the "Breaking Changes" section - before this PR, it's "Upcoming", but when this PR goes in, that's when it will be _actually_ breaking

Reply:
Okay, I'm working on it.